ros_gz_sim/setup.cfg

```diff
@@ -0,0 +1,3 @@
+[options.entry_points]
+launch.frontend.launch_extension =
+  ros_gz_sim = ros_gz_sim
```

Comment:
nit: end file with a newline (see https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
Added in [2416aaf](https://github.com/gazebosim/ros_gz/pull/528/commits/2416aafce0bd0b4e3ff2f1ca3ecfdb3e4d2813cf).